std/format.d

```diff
@@ -1120,8 +1121,8 @@ struct FormatSpec(Char)
             {
                 if (trailing.ptr[0] == ' ')
                 {
-                    while (!r.empty && std.ascii.isWhite(r.front)) r.popFront();
-                    //r = std.algorithm.find!(not!(std.ascii.isWhite))(r);
```

Comment:
I'd suggest that in the cases where both std.utf and std.ascii are being used, it would be better to use static imports and use the full import path; otherwise, I'd be worried about someone accidentally changing which functions got called when they refactored the code - especially if both work for the code in question, but one has worse performance.